stable/metrics/values.yaml

```diff
@@ -31,14 +31,18 @@ conf:
   sqsPollingInterval: 10
   checksQueueArn: arn:aws:sqs:local:TBD:TBD
   scansQueueArn: arn:aws:sqs:local:TBD:TBD
+  findingsQueueArn: arn:aws:sqs:local:TBD:TBD
```

Comment:
Please, declare the new findingsQueueArn in stable/vulcan/values.yaml:

-  In the metrics section
- In the goaws.yaml section

Reply:
It should be done now.